ArduCopter/events.cpp

```diff
@@ -430,9 +432,9 @@ void Copter::set_mode_auto_do_land_start_or_RTL(ModeReason reason)
         AP_Notify::events.failsafe_mode_change = 1;
         return;
     }
+    GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "AutoRTL Unavailable, Trying RTL Mode");
```

Comment:
Can't move these inside the #endif - the idea is that the message is emitted whenever it can't enter the mode (AutoRTL here), regardless of why.  So if it fails to enter AutoRTL because there's no GPS available then we should still emit the message.